glyphs-reader/src/font.rs

```diff
@@ -1068,9 +1148,33 @@ impl TryFrom<RawFont> for Font {
         };
         let units_per_em = units_per_em.try_into().map_err(Error::InvalidUpem)?;
 
+        let mut names = BTreeMap::new();
+        for name in from.properties.unwrap_or_default() {
+            // TODO: we only support dflt, .glyphs l10n names are ignored
+            name.value
+                .or_else(|| {
+                    name.values
+                        .unwrap_or_default()
+                        .iter()
+                        .find(|v| v.language == "dflt")
+                        .map(|v| v.value.clone())
+                })
+                .and_then(|value| names.insert(name.key, value));
+        }
+        names.insert("familyNames".into(), from.family_name);
+        let version_string = names.remove("versionString");
+        let empty_string = String::from("");
+        let version = format!(
+            "{}.{:0>3}{}{}",
+            from.versionMajor.unwrap_or_default(),
+            from.versionMinor.unwrap_or_default(),
+            version_string.as_ref().map(|_| "; ").unwrap_or(""),
+            version_string.as_ref().unwrap_or(&empty_string),
```

Comment:
where did you infer this treatment for versionString? glyphsLib doesn't seem to be doing this, but simply setting whatever is the versionString to the corresponding UFO's openTypeNameVersion (which will become nameID=5); whereas here you are appending it after a semicolor to the version string generated from major/minor.

Reply:
I made it up based on the ability to set a version - which fits the OT "Should begin with the syntax “Version <number>.<number>” (upper case, lower case, or mixed, with a space between “Version” and the number)." perfectly - and a version string in the UI. However, upon test exporting from a font with both version and version string set name 5 is just the version string and the values I punched for version are just gone.

Reply:
> name 5 is just the version string and the values I punched for version are just gone.

yep, versionString defines the whole nameID=5

Reply:
Testing w/Glyphs App suggests if version string is there, just use it. If not, build the version string from minor/major. Will add a test for this.